BlackJack.py

```diff
@@ -0,0 +1,147 @@
+from random import choice
+from enum import Enum, auto
```

Comment:
musisz zmienic na aenum: [https://stackoverflow.com/a/35968057] , albo rozkminic to inaczej... niestety JAVOWE enumy się różnią od tych w pythonie